subgraph/README.md

```diff
@@ -0,0 +1,80 @@
+# The Graph Integration
+
+This directory contains all documents and files related to the NFA x The Graph integration.
+
+## What is the Graph?
+
+Per the official documentation:
+
+> The Graph is a decentralized protocol for indexing and querying data from blockchains, starting with Ethereum. It makes it possible to query data that is difficult to query directly.
+
+In short, the Graph offers its users a flexible and almost instantaneous way to query data from blockchains with the integration of subgraphs.
+
+Subgraphs are APIs that can be queried with a standard GraphQL API. Each subgraph's manifest describes the data structure of contracts of interest and how they should be indexed. The current directory contains the subgraph implementation for the Fleek NFA contract.
+
+## Directory Structure
+
+All of the files and directories except one are auto-generated by the `graph-cli` package (as you will learn in the next section) based on the current live version of the contract on Polygon's mumbai testnet.
+
+**Directories:**
+
+- [abis](./abis/): contains all ABIs related to all contracts of interest in the subgraph with their JSON format. These ABI files are all generated based on the verified version of the contract on the blockchain.
+- [build](./build/): contains the latest build of the subgraph with its WASM (which is ignored in the repository).
```

Comment:
All generated by graph cli. Also, I have not added any ABI to this subgraph folder (or any). They're generated by graph-cli as needed.

The build folder is what is uploaded to IPFS and then used for the subgraph deployment, so it makes sense to have the ABI here (even though it's really not in our hands to decide what it should contain or shouldn't since it is generated)

The schema and the yaml files are both based on the versions in the root of the subgraph directory, and also autogenerated.

The yaml and schema files at the root are written by me.

Reply:
Maybe we could do this

- add to .gitignore (in this PR)
- in CI, do the build, and add some logs what we can (in another PR)